src/main.cpp

```diff
@@ -76,8 +78,21 @@ class MetaGenerationConsumer : public clang::ASTConsumer {
         // Generate TypeScript definitions
         if (!cla_outputDtsFolder.empty()) {
             llvm::sys::fs::create_directories(cla_outputDtsFolder);
+
+            // find iOS SDK docset path
+            std::string docSetPath;
+            if (!cla_docSetFile.empty()) {
+                docSetPath = cla_docSetFile.getValue();
+            }
+            else if (struct passwd* pwd = getpwuid(getuid())) {
```

Comment:
I rather meant that we should not hardcode to the iOS docset so as to be able to support tvOS, watchOS and macosx down the line. Maybe `EFFECTIVE_PLATFORM_NAME ` is not the correct build setting because of the simulator thing and in that case we should find a way to correctly infer the docset we need.